MSALiOSB2C/ViewController.swift

```diff
@@ -353,26 +292,39 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
     override func viewWillAppear(_ animated: Bool) {
         
         if self.accessToken.isEmpty {
-            
-            signoutButton.isEnabled = false;
-            callGraphApiButton.isEnabled = false;
-            editProfileButton.isEnabled = false;
-            refreshTokenButton.isEnabled = false;
-            
-            
+            signoutButton.isEnabled = false
+            callGraphApiButton.isEnabled = false
+            editProfileButton.isEnabled = false
+            refreshTokenButton.isEnabled = false
         }
     }
     
-    func getUserByPolicy (withUsers: [MSALUser], forPolicy: String) throws -> MSALUser? {
+    func getAccountByPolicy (withAccounts: [MSALAccount], policy: String) throws -> MSALAccount? {
```

Comment:
nit: let's have the parameter called just accounts? We can still use withAccounts as a label, e.g.

`func getAccountByPolicy(withAccounts accounts: MSALAccount) { ... }`

Reply:
:+1: cool, changed